src/ui/components/Timeline.qml

```diff
@@ -325,6 +325,16 @@ Item {
                     ]);
                 }
             }
+            Action {
+                id: debiasAction;
+                icon.name: "bias";
+                text: qsTr("Estimate gyro bias here");
+                onTriggered: {
+                    const pos = root.position;
+
+                    controller.estimate_bias(pos);

```

Comment:
This is short enough to be on a single line `onTriggered: controller.estimate_bias(root.position);`